route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -46,7 +46,8 @@ namespace route_following_plugin
         
         pnh_->param<double>("minimal_maneuver_duration", mvr_duration_, 16.0);
         pnh2_->param<double>("config_speed_limit",config_limit);
-        pnh_->param<double>("/guidance/route_end_jerk", jerk_, 0.05);
+        pnh_->param<double>("buffer_time_lanechange",buffer_lanechange_time_);
+        pnh_->param<double>("/guidance/route_end_jerk", jerk_, 1.0);
```

Comment:
Not strictly a defect, but I'm not sure a private node handle should be referencing parameters starting with `/`, this makes it really hard to remap this parameter in a launch file if we need to. I can make a note of this to sweep up in another story or something.